README.md

```diff
@@ -6,6 +6,38 @@ Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
 
 ---
+## Getting started 
+These instructions will get you a copy of the project up and running in your local machine for development and testing purposes.
+
+## Prerequisites
+- [Git](https://git-scm.com/download/)
+- [Python 3 and above](https://www.python.org/downloads/)
+- [PostgreSQL](https://www.postgresql.org/)
+
+## Installing
+- Create a working space in your local machine
+- Clone this [repository](https://github.com/andela/ah-django-unchained) `git clone https://github.com/andela/ah-django-unchained.git`
+- Navigate to the project directory
+- Create a virtual environment `python3 -m venv/path/to/new/virtual/environment`
+- Activate your virtual environment `source name_of_your_virtual_environment/bin/activate`
+- Install dependencies to your virtual environment `pip install -r requirements.txt`
+
+### Setting up the database
+- Start your database server and create your database
+- Create a .env file and put these key=values in it:
```

Comment:
I feel that setting up the .env file should be under Installation sub-header as it does not have any relationship with setting up the database. It should appear after setting up the virtual environment so that when you do your requirements installation, it is done when your virtual environment is active.

Reply:
Thanks Ken. We have worked on that.